src/components/ReactTags.js

```diff
@@ -131,13 +131,17 @@ class ReactTags extends Component {
       return this.props.handleFilterSuggestions(query, suggestions);
     }
 
-    return suggestions.filter((item) => {
-      return (
-        item[this.props.labelField]
-          .toLowerCase()
-          .indexOf(query.toLowerCase()) === 0
-      );
+    const queryIndex = (item) =>
```

Comment:
can you move this to an instance function `getQueryIndex` ?

Reply:
done